### ISSUE TITLE
Reflection warning in v0.4.0

### ISSUE THREAD

**Issue by BrunoBonacci** [clojure-emacs/clj-suitable#19]
please fix the following reflection warning.

```
Reflection warning, cider/nrepl/inlined_deps/suitable/v0v4v0/suitable/complete_for_nrepl.clj:112:6 - reference to field getName can't be resolved.
```

**Comment by vemv** [comment]
Released `[org.rksm/suitable "0.4.1-alpha1"]`, feel free to try it out

**Comment by vemv** [comment]
Whoops, sorry, it's not as easy as bumping that dep isolatedly. We have to cut a cider-nrepl release that uses the new `suitable`.

**Comment by vemv** [comment]
Fix finally available in `cider-20211002.1153.tar` (i.e. today's CIDER snapshot published on MELPA) or via `[org.rksm/suitable "0.4.1"]` for other setups
